packages/babel-template/package.json

```diff
@@ -9,7 +9,6 @@
   "main": "lib/index.js",
   "dependencies": {
     "babel-code-frame": "7.0.0-beta.3",
-    "babel-traverse": "7.0.0-beta.3",
```

Comment:
This severs the cyclic dependency that I filed. By making `babel-template` only depend on `babel-types`, we avoid risking cycles that we'd otherwise get if we split things out of `babel-traverse` but still wanted to use them as dependencies in `babel-traverse`.